flocker/ca/__init__.py

```diff
@@ -9,7 +9,8 @@
     "ComparableKeyPair", "PathError", "CertificateAlreadyExistsError",
     "KeyAlreadyExistsError", "EXPIRY_20_YEARS",
     "AUTHORITY_CERTIFICATE_FILENAME", "AUTHORITY_KEY_FILENAME",
-    "CONTROL_CERTIFICATE_FILENAME", "CONTROL_KEY_FILENAME"
+    "CONTROL_CERTIFICATE_FILENAME", "CONTROL_KEY_FILENAME",
+    "DEFAULT_CERTIFICATE_PATH"
```

Comment:
This should be in `flocker/control/script.py` - see discussion in another comment.